packages/build-tools/src/utils/project.ts

```diff
@@ -159,3 +194,11 @@ async function runExpoDoctor<TJob extends Job>(ctx: BuildContext<TJob>): Promise
     }
   }
 }
+
+export function readPackageJson(projectDir: string): any {
+  const packageJsonPath = path.join(projectDir, 'package.json');
+  if (!fs.pathExistsSync(packageJsonPath)) {
+    throw new Error(`package.json does not exist in ${projectDir}`);
+  }
+  return fs.readJSONSync(packageJsonPath);
```

Comment:
```suggestion
  try {
    return fs.readJSONSync(packageJsonPath);
  } catch (err) {
    throw new Error(`Failed to parse or read package.json: ${err.message}`);
  }
```
maybe? As far as I remember readJson on it's own will just return a message unexpected symbol at line x without any context